app/templates/layouts/asylum-base.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="charset" content="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>{% block page_title %}{{ config.APP_NAME }}{% endblock %}</title>
+
+        {% assets 'asylum_css' %}<link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}">{% endassets %}
+        {% assets 'asylum_scss' %}<link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}">{% endassets %}
+
+        <!-- Google Translate -->
+        <script src="http://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
```

Comment:
totally arbitrary. it happened because the original asylum code put the translate script in `<head>` and the map script at the top of `<body>` so i just kept that